src/v/cluster/controller_backend.cc

```diff
@@ -561,8 +561,35 @@ ss::future<> controller_backend::fetch_deltas() {
           return ss::with_semaphore(
             _topics_sem, 1, [this, deltas = std::move(deltas)]() mutable {
                 for (auto& d : deltas) {
+                    vlog(clusterlog.trace, "got delta: {}", d);
```

Comment:
nit: i think this comes from debugging, if it is required please prefix with `[{}]` ntp

Reply:
I think it should stay, it helps with ci-failure investigations. I didn't add `[{}]` because the ntp is printed as part of delta itself.